test/dynamo/test_fwd_loss_bwd.py

```diff
@@ -0,0 +1,253 @@
+# Owner(s): ["module: dynamo"]
+
+import torch
+from torch._dynamo.testing import CompileCounterWithBackend, normalize_gm
+from torch.testing._internal.common_utils import run_tests, TestCase
+
+
+class TestForwardLossBackward(TestCase):
+    def test_autograd_grad_basic(self):
+        mod = torch.nn.Linear(4, 4)
+        x = torch.randn(2, 4)
+
+        def fn(x):
+            res = mod(x)
+            loss = res.sum()
+            params = tuple(mod.parameters())
+            grads = torch.autograd.grad(loss, params)
+            return loss.detach(), grads[0].sum(), grads[1].sum()
+
+        cnt = CompileCounterWithBackend("aot_eager")
```

Comment:
Use `AOTEagerAndRecordGraphs` backend and do an ExpectedInline on aot_eager as well. This will show that aot dispatcher is tracing through the autograd.grad.